src/Sulu/Bundle/SecurityBundle/Tests/Functional/Command/CreateUserCommandTest.php

```diff
@@ -18,10 +18,8 @@
 
 class CreateUserCommandTest extends SuluTestCase
 {
-    /**
-     * @var CommandTester
```

Comment:
Still, we always write these annotations, and it is much more comfortable in the IDE if you have them.